src/meta/app/src/principal/tenant_user_ident.rs

```diff
@@ -70,7 +70,11 @@ mod tests {
 
     fn test_format_parse(user: &str, host: &str, expect: &str) {
         let tenant = Tenant::new_literal("test_tenant");
-        let user_ident = UserIdentity::new(user, host);
+        let user_ident = UserIdentity {
```

Comment:
test_tenant_user_ident_as_kvapi_key use this function. So use struct init replace new function.

Reply:
cc @drmingdrmer